src/InteractionEvents33.jsx

```diff
@@ -0,0 +1,18 @@
+export default function InteractionEvents33() {
+  function handlerClick() {
+    alert("Button was clicked.");
+  }
+
+  function handlerChange() {
```

Comment:
dar dėl tikslumo. Šios funkcijos dažniausiai vadinamos `handleClick, handleChange`, be r raidės. Tai labai dažnai naudojamas pavadinimas, tai geriau jo laikytis tiksliai :) Arba būna `clickHandler, changeHandler`- tada su r.